dev/ci/cases/sfs/C192mx025_S2S_CPC_ICS.yaml

```diff
@@ -0,0 +1,19 @@
+experiment:
```

Comment:
Neil, These new case files look strong (along with their overwriting defaults).  Do you have these **sfs** cases tested in automated scripts yet?

Reply:
What framework will you be using for automation?  global-workflow is currently developing framework in VLab's GitLab that uses these case files.

Reply:
@TerrenceMcGuinness-NOAA I'm not sure if this answers your question, but to run the full set of SFS runs, which is currently November 1st and May 1st start dates, we add 

<cycledef group='"gefs"'>${YEAR}${MONTH}010000 ${YEAR}${MONTH}010000 24:00:00</cycledef>

to the rocoto .xml file before the rocoto run as rocoto does not support (to my knowledge) starting once a month on the first.